packages/chrome.extensions.vm/chrome.extensions.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>chrome.extensions.vm</id>
-    <version>0.0.0.20250123</version>
+    <version>0.0.0.20250217</version>
     <authors>Mandiant</authors>
     <description>A package for multiple useful chrome extensions from the Chrome webstore.</description>
     <dependencies>
-      <dependency id="common.vm" />
+      <dependency id="common.vm"  version="0.0.0.20250206" />
       <dependency id="googlechrome.vm" />
     </dependencies>
-  </metadata>
+    <tags>Productivity Tools</tags>
+</metadata>
```

It seems you have changed the indentation by accident in all the nuspecs
```suggestion
  </metadata>
```

---

this issue is still not resolved

---

my bad, I didn't update the script, it is fixed now